configs/fedora.toml

```diff
@@ -0,0 +1,78 @@
+# A basic configuration for Fedora's message broker, using the example callback
+# which simply prints messages to standard output.
+#
+# This file is in the TOML format.
+amqp_url = "amqps://fedora:@rabbitmq.fedoraproject.org/%2Fpublic_pubsub"
+passive_declares = false
```

Comment:
Shouldn't that be `true`?

Reply:
No, it needs to be false because in the public_pubsub vhost users  create their own queues. Since the default is false I think this should be dropped here, though. Looked like I dropped it from the stg config, but not here.

Reply:
Right, I mixed it up. Do you think there may be errors trying to declare the exchange?

Reply:
It works fine in staging. I'm having an issue in production that looks like a proxy problem - the initial TCP handshake isn't completing, but it's the same configuration.